### ISSUE TITLE
Error on import

### ISSUE THREAD

**Issue by fdufnews** [nevyn-hira/macrokeyboard#1]
Hi,
I have the following error when I launch macrokeyboard

> frederic@frederic-Desktop:~/Tools/macrokeyboard$ ./macrokeyboard -v
Traceback (most recent call last):
  File "/home/frederic/Tools/macrokeyboard/./macrokeyboard", line 11, in <module>
    import notify2 as notify
ModuleNotFoundError: No module named 'notify2'

pynotifier is installed but there is no notify2 anywere in

running Python 3.10.12

**Comment by fdufnews** [comment]
Found the problem.
One need to make
>pip3 install notify2

But now I have other errors. I investigate and will come back if necessary.

Edit: The remaining errors were caused by an incomplete json file.

It will be interesting to have a little more details on what is necessary in the json file and what are the key pair for

**Comment by fdufnews** [comment]
Forget this issue.
What a pity, I have too many problems to make it work with Wayland.

**Comment by nevyn-hira** [comment]
Oh hey! Someone is actually using it. Sorry. I hadn't really thought anyone was using it.

It will not work with Wayland at the moment. I don't use Gnome or KDE/Plasma so I haven't made the switch yet BUT I was just looking at the code tonight thinking it needs a serious refresh (a lot of the software I rely on nowadays is being run from flatpak but the code, as it's written now, doesn't really allow for command line parameters in the "switchTo" configuration.

I was debating on whether to rewrite the code in golang (VERY fast compiled language), also fixing up its shortcomings OR writing something up in python/tkinter to edit the configuration file.

In the end I decided on neither BUT if you're still wanting to use this, I'd be quite happy to make this work in Wayland.

**Comment by fdufnews** [comment]
Thank you very much for your suggestion but I have since made my own Macropad based on a Raspberry Pi Pico.

**Comment by nevyn-hira** [comment]
Closing issue. Have updated the readme file with the dependency.

I have looked into the other things discussed here. Can not fix as Wayland is missing the accessibility features that this code relies on. A simpler version of this code could be made for Wayland with no window management which is probably not far off what a lot of people want anyway. It's just that my usage is around the window management.
